packages/react-server-dom-parcel/static.node.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export {prerenderToNodeStream} from './src/server/react-flight-dom-server.node';
```

Comment:
These have changed names now but nbd. I'll take a pass over to make sure all changes are caught up once we land.